src/components/MainIcon/MainIcon.tsx

```diff
@@ -0,0 +1,57 @@
+import { useRecoilValue } from "recoil";
+
+import {
+  TodoIconPositionState,
+  MessengerIconPositionState,
+  clickedIconState,
+} from "../../states/atom";
```

Comment:
```suggestion
import {
  todoIconPositionState,
  messengerIconPositionState,
  clickedIconState,
} from "../../states/atom";
```

저라면 일반 state는 파스칼케이스보단 카멜케이스를 쓸 것 같아요!!
보통 컴포넌트명을 파스칼케이스로 쓰니까 상태명에도 파스칼케이스를 쓰면
약간 혼돈의 여지가 있을 것 같아요:)

Reply:
앗 네...! 고치도록 하겠습니다!